File: go-sdk/pkg/models/shared/webhooks.go
package shared

// Webhooks
// The part of an API response that describes marker
// based pagination
type Webhooks struct {
	Entries    []WebhookMini `json:"entries,omitempty"`
	Limit      *int64        `json:"limit,omitempty"`
	NextMarker *int64        `json:"next_marker,omitempty"`
	PrevMarker *int64        `json:"prev_marker,omitempty"`
}

File: sdks/python/src/boxsdk/models/shared/webhooks.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from boxsdk import utils
from ..shared import webhook_mini as shared_webhook_mini


@dataclass_json
@dataclasses.dataclass
class Webhooks:
    r"""Webhooks
    The part of an API response that describes marker
    based pagination
    """
    
    entries: Optional[list[shared_webhook_mini.WebhookMini]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    prev_marker: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('prev_marker') }})
    
